src/analysis/retail/hunter/marksmanship/modules/resources/MarksmanshipFocusCapTracker.tsx

```diff
@@ -0,0 +1,16 @@
+import { TRUESHOT_FOCUS_INCREASE } from 'analysis/retail/hunter/marksmanship/constants';
+import { FocusCapTracker } from 'analysis/retail/hunter/shared';
+import { HUNTER_BASE_FOCUS_REGEN } from 'analysis/retail/hunter/shared/constants';
+import SPELLS from 'common/SPELLS';
+
+class MarksmanshipFocusCapTracker extends FocusCapTracker {
+  getBaseRegenRate() {
+    let regenRate = HUNTER_BASE_FOCUS_REGEN;
+    if (this.selectedCombatant.hasBuff(SPELLS.TRUESHOT.id)) {
```

Comment:
Why did you add this file back after I removed it in https://github.com/WoWAnalyzer/WoWAnalyzer/pull/7394 ?